website/src/categories/components/icon/icon.md

```diff
@@ -11,6 +11,6 @@ packageName: icon
 eleventyNavigation:
   parent: Components
   key: Icon
-  order: 11
+  order: 14
   status: coming
```

Comment:
Images for form and form field are missing? :)

<img width="639" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/3f61905f-1a8e-4237-9cbe-4564e4618bad">